index.md

```diff
@@ -1,5 +1,9 @@
+---
+relatedlinks: "[Juju &nbsp; ecosystem &nbsp; docs](https://juju.is/docs), [Juju &nbsp; docs](https://documentation.ubuntu.com/juju/), [JAAS &nbsp; docs](https://documentation.ubuntu.com/jaas/), [Jubilant &nbsp; docs](https://documentation.ubuntu.com/jubilant/), [Charmcraft &nbsp; docs](https://documentation.ubuntu.com/charmcraft/), [Ops &nbsp; docs](https://documentation.ubuntu.com/ops/), [Charmlibs &nbsp; docs](https://canonical-charmlibs.readthedocs-hosted.com/)"
```

Comment:
This one links to JAAS but not Terraform.

Reply:
Another oops. My brain's fried, sorry. Fixed it now (with `commit --amend` -- I was hoping I'd fix it before you'd get to it and see how I've blundered again :blush: )